plugins/pipelines/jobBadge.js

```diff
@@ -62,6 +62,17 @@ module.exports = config => ({
                     return reply.redirect(getUrl(badgeConfig));
                 }
 
+                if (job.state === 'DISABLED') {
+                    return reply.redirect(getUrl(Object.assign(
```

Comment:
can we still fetch the last build status and display it

`pipeline/job success` in grey color

Reply:
i thinks this is confusing by mixing the color and words